main.js

```diff
@@ -1,19 +1,148 @@
+}
+
+document.querySelector('.card-name-3').textContent = projectData[2].name;
+document.querySelector('.card-short-3').textContent = projectData[2].short;
+document.querySelector('.card-img-3').src = projectData[2].imgSrc;
+for (let i = 0; i < projectData[2].techUsed.length; i += 1) {
+  const newLi = document.createElement('li');
+  newLi.textContent = projectData[2].techUsed[i];
+  document.querySelector('.card-tech-3').appendChild(newLi);
+}
+
+document.querySelector('.card-name-4').textContent = projectData[3].name;
+document.querySelector('.card-short-4').textContent = projectData[3].short;
+document.querySelector('.card-img-4').src = projectData[3].imgSrc;
+for (let i = 0; i < projectData[3].techUsed.length; i += 1) {
+  const newLi = document.createElement('li');
+  newLi.textContent = projectData[3].techUsed[i];
+  document.querySelector('.card-tech-4').appendChild(newLi);
+}
+
+// Show details
+
+const detailsButton1 = document.querySelector('.details-button-1');
+const detailsButton2 = document.querySelector('.details-button-2');
+const detailsButton3 = document.querySelector('.details-button-3');
+const detailsButton4 = document.querySelector('.details-button-4');
+
+function showDetails(e) {
+  document.querySelector('.app-details').classList.replace('section-hide', 'section-show');
+  document.querySelector('.details-title').textContent = projectData[e.currentTarget.numpro].name;
+  document.querySelector('.details-desc').textContent = projectData[e.currentTarget.numpro].desc;
+  document.querySelector('.details-img').src = projectData[e.currentTarget.numpro].imgSrc;
+  const techList = document.querySelector('.details-tech');
+  const numLi = techList.children.length;
+  if (numLi > 0) {
+    for (let i = 0; i < numLi; i += 1) {
+      techList.firstElementChild.remove();
+    }
+  }
+  for (let i = 0; i < projectData[e.currentTarget.numpro].techUsed.length; i += 1) {
+    const newLi = document.createElement('li');
+    newLi.textContent = projectData[e.currentTarget.numpro].techUsed[i];
+    techList.appendChild(newLi);
+  }
+  document.querySelector('.details-liveLink').href = projectData[e.currentTarget.numpro].liveLink;
+  document.querySelector('.details-sourceLink').href = projectData[e.currentTarget.numpro].sourceLink;
+}
+
+detailsButton1.addEventListener('click', showDetails);
+detailsButton1.numpro = 0;
+
+detailsButton2.addEventListener('click', showDetails);
+detailsButton2.numpro = 1;
+
+detailsButton3.addEventListener('click', showDetails);
+detailsButton3.numpro = 2;
+
+detailsButton4.addEventListener('click', showDetails);
+detailsButton4.numpro = 3;
+
+// Hide details
+
+const closeDetails = document.querySelector('.close-details');
+
+function hideDetails() {
+  document.querySelector('.app-details').classList.replace('section-show', 'section-hide');
 }
 
-burgerMenu.addEventListener('click', showMenu);
\ No newline at end of file
+closeDetails.addEventListener('click', hideDetails);
```

Comment:
- [ ] As stated by the previous reviewer, Kindly fix your design in mobile version @375px to the problem which is the close icon is hidden, so I can't close the popup.
> Your design
![image](https://user-images.githubusercontent.com/53402600/195465793-b132ce7f-46ec-46dc-a0be-5e099477ff0f.png)

Reply:
Hi, thank you for your kind response, I tried to replicate the error in Chrome with the dimensions you are mentioning but I still cannot see it, I've tried also with Opera, Firefox, and Edge, so I'm really confused about it, but still I've made some changes so the card should stick to the top in any case and you can see the button.

![Screenshot 2022-10-12 201421](https://user-images.githubusercontent.com/44882884/195476309-242b48f9-f773-43e7-8640-46f2065416dd.jpg)